hl_extractor/hl_calc.py

```diff
@@ -47,7 +48,7 @@ def _calculate(self):
             f.write(self.ll_data.encode("utf-8"))
             f.close()
         except IOError:
-            print("IO Error while writing temp file")
+            current_app.logger.error("IO Error while writing temp file")
```

Comment:
In further testing, I found that this is a problem. access to `current_app` isn't thread-safe, even though the underlying logging object is. I'll revert the changes in this file, pending some changes that I'm making in #378